msal/versioning/version.properties

```diff
@@ -1,3 +1,3 @@
 #Wed Aug 01 15:24:11 PDT 2018
-versionName=5.3.1
+versionName=5.4.0
```

Comment:
Should these versions be updated as part of these changes? Same applies for the changes in `msalautomationapp/build.gradle`, and `msal/build.gradle

Reply:
I think the changes is because of the merging with the dev or fadi/release/branch